ale_linters/sass/sasslint.vim

```diff
@@ -3,6 +3,6 @@
 call ale#linter#Define('sass', {
 \   'name': 'sasslint',
 \   'executable': 'sass-lint',
-\   'command': 'sass-lint -v -q -f compact %t',
```

Comment:
That won't work on Windows. Look at how other linters use `BufferCdString`. `grep` for it.